tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -194,13 +248,13 @@ static void le_security_changed(struct bt_conn *conn, bt_security_t level,
 		 * This means bond is lost and we restart pairing to re-bond
 		 */
 		if (bt_conn_get_info(conn, &info) == 0 &&
-		    info.role == BT_CONN_ROLE_CENTRAL) {
+			info.role == BT_CONN_ROLE_CENTRAL) {
 			LOG_DBG("Bond lost");
 
 			bt_addr_le_copy(&bond_ev.address, addr);
 
 			tester_event(BTP_SERVICE_ID_GAP, BTP_GAP_EV_BOND_LOST,
-				     &bond_ev, sizeof(bond_ev));
+					 &bond_ev, sizeof(bond_ev));
```

Comment:
Same here. Even if there was a need to fix coding style issues such changes should not be mixed into commits which contain functional changes.